django/db/backends/mysql/features.py

```diff
@@ -33,6 +33,20 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_slicing_ordering_in_compound = True
     supports_index_on_text_field = False
     has_case_insensitive_like = False
+    procedure_without_params_sql = """
+        CREATE PROCEDURE test_procedure ()
+        BEGIN
+            DECLARE V_I INTEGER;
+            SET V_I = 1;
+        END;
+    //"""
```

Comment:
It won't work properly without `//` (or `/` on Oracle) if you try to concatenate both `procedure_without_params_sql + procedure_with_int_param_sql` (or add others DML) and execute. In this specific usage we can get rid of them.